clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HostLevelIngestionStats.java

```diff
@@ -325,6 +327,12 @@ public HostLevelIngestionStats(
         () -> totalStats.ingestionFailureSensor,
         new Count());
 
+    this.resubscriptionFailureSensor = registerPerStoreAndTotalSensor(
+        "resubscription_failure",
+        totalStats,
+        () -> totalStats.resubscriptionFailureSensor,
+        new Count());
```

Comment:
FYI, Count metric would always increase and doesn't reset until we bounce the node, but maybe this is what we want here?

Reply:
Cumulative failure count is sth we expect to have here.